vcd/resource_vcd_vapp_vm_test.go

```diff
@@ -156,7 +156,8 @@ resource "vcd_vapp_vm" "{{.VmName}}" {
   catalog_name  = "{{.Catalog}}"
   template_name = "{{.CatalogItem}}"
   memory        = 1024
-  cpus          = 1
+  cpus          = 2
+  cpu_cores		= 1
```

Comment:
Not sure if it's github or the "= 1" part is misaligned

Reply:
Github issue, as pasted after `terraform fmt`

Reply:
It's not a github issue. It is space vs tabs. Go fmt does not convert from one into the other which results into this.

Reply:
Looked in wrong place -fixed now.